### ISSUE TITLE
Model averaging description table

### ISSUE THREAD

**Issue by Sikys11235** [PetrCala/Diploma-Thesis#37]
When trying to run the code exporting description table (around line 620 of the main script), received this error (not sure if it is just me)
Error in data.frame(variable = as.character(var_verbose), description = as.character(var_desc),  : 
  arguments imply differing number of rows: 1, 0
In addition: Warning message:
Unknown or uninitialised column: `var_name_description`. 

It works fine after adding the line right before the call (I assume the error is just a mismatched name)
######Modified part#######
var_list$var_name_description <- var_list$Description
#MA variable description table
...(etc)

**Comment by PetrCala** [comment]
Yeah, it seems to be a name mismatch. In the readme for the `getMAVariablesDescriptionTable` function, you can see the return object description

```R
#' @return [data.frame] A data frame that contains the following columns: 'Variable', 
#' Description', 'Mean', 'SD'. 'Variable' corresponds to 'var_name_verbose' from the 'input_var_list', 
#' Description' corresponds to 'var_name_description' from the 'input_var_list',
#' 'Mean' and 'SD' are the mean and standard deviation of the variable in 'bma_data'.
```

So the original name, `var_name_description`, is wrongly expected in the return object of the `getMAVariablesDescriptionTable`, where `Description` is used instead.

I'll try to get it cleaned up, thanks for the heads up.
